README.md

```diff
@@ -14,99 +15,186 @@ Keep reading to get started or check out the [Roadmap](#roadmap) to see the feat
 
 ## Getting Started
 
-1. Clone this repo, `cd` into it, and `yarn install`
+### Step 1
 
-```
+Clone this repo, `cd` into it, and `yarn install`
+
+```zsh
 git clone git@github.com:redwoodjs/example-store-stripe.git
-cd example-store
+cd example-store-stripe
 yarn install
 ```
 
-2. Get your Stripe test keys
+### Step 2
+
+Get your Stripe test keys
 
 To develop on this repo locally, you'll need to populate your `.env` file with a few env vars. The first of those is your Stripe test keys.
 
-You'll need a Stripe account to get your test keys.
-If you don't already have one, you can make one here: https://dashboard.stripe.com/login?redirect=%2Ftest%2Fdashboard.
+First time visitor gets greeted like:
```

Comment:
First time visitors to what? I know you know, but there's no easy link to click for users anymore with this change. We should probably add it back, and it looks like you want to link to the homepage now if I'm not mistaken:
```suggestion
First time visitors to [Stripe](https://stripe.com/) gets greeted like:
```